root/components/home/Intro.js

```diff
@@ -0,0 +1,265 @@
+            <div className="row">
+              <div
+                className="col-lg-9 d-flex flex-column"
+              >
+                <h1
+                  className="ms-4 mt-4"
+                  style={{color: 'white', height: '5vh'}}
+                >
+                  {l('The Music Database')}
+                </h1>
+                <h2
+                  className="ms-4 mt-2"
+                  style={{color: 'white', height: '5vh'}}
+                >
+                  {l(`World's Biggest Open Source Music Database`)}
+                </h2>
+                <div className="row mt-2 ms-2 mb-2">
+                  <div className="col-8 col-md-10">
+                    <input
+                      className="form-control special-font"
+                      id="searchInputMain"
+                      name="query"
+                      onKeyDown={event => {
+                        if (event.key === 'Enter') {
+                          const query =
+                              document.getElementById('searchInputMain');
+                          console.log(query.value);
+                          if (query.value.trim().length < 1) {
+                            return false;
+                          }
+                          let searchType;
+                          if (typeCurrent === 'CD Stub') {
+                            searchType = 'cdstub';
+                          } else if (typeCurrent === 'Documentation') {
+                            searchType = 'doc';
+                          } else {
+                            searchType =
+                                typeCurrent.replace(' ', '_').toLowerCase();
+                          }
+                          window.open('https://musicbrainz.org/' + 'search?type=' + searchType + '&query=' + query.value, '_self');
+                          return false;
+                        }
+                        return false;
+                      }}
+                      placeholder="Search 41,054,421 Entities"
+                      style={{textTransform: 'capitalize'}}
+                      type="search"
+                    />
+                  </div>
+                  <div className="col-4 col-md-2">
+                    <button
+                      className="btn btn-primary"
+                      onClick={searchButtonClick}
+                      type="button"
+                    >
+                      <FontAwesomeIcon
+                        icon={faSearch}
+                        size="lg"
+                      />
+                    </button>
+                    <button
+                      className="btn btn-b-n"
+                      onClick={this.handleShow}
+                      type="button"
+                    >
+                      <FontAwesomeIcon
+                        icon={faScanner}
+                        size="lg"
+                      />
+                    </button>
```

Comment:
This button does not show on test at the moment so I'm not sure what the point is supposed to be (https://fontawesome.com/icons/scanner?s=solid ?) but I'm confused about needing a second button other than "search".

Edit: Oh, I understand reading a few lines later that the idea is to have a way to scan a barcode - is this something anyone has expressed an interest on? I'm a bit confused by the addition.

Reply:
Try out the button placed on https://musicbrainz-web.web.app @reosarevok

Reply:
Removed this for now.